frontend/src/components/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChevronRight as ChevronRightIcon } from '@mui/icons-material';
+import { Box, Link, Paper, Table, TableBody, TableCell, TableRow, Typography } from '@mui/material';
+import Router from 'next/router';
+import React from 'react';
+
+interface Props {
+  data: {
+    title: string;
+    description: string;
+    table: {
+      item: string;
+      value: string;
+      link: string;
+    }[];
+  }
+}
+
+function InfoCard({ data }: Props) {
+  const handleClick = (link: string) => {
+    try {
+      Router.push(link);
+    } catch(err){
+      console.error("Not Exist Page")
+    }
+  }
+
+  return (
+    <Paper sx={{minWidth: 723, maxWidth: 1023}}>
+      <Box sx={{minWidth:680, pt: 3, px: 2, pb: 1}}>
+        <Typography variant="h5">
+          {data.title}
+        </Typography>
+        <Typography variant="body2" color="text.secondary" sx={{pt: 1}}>
```

Comment:
```suggestion
        {data.description ? (<Typography variant="body2" color="text.secondary" sx={{pt: 1}}>
```
デスクリプションがない時用処理を三項演算子で書いときたい